src/tic_tac_toe/referee.clj

```diff
@@ -11,3 +11,36 @@
 (defn validate-move
   [player move]
   (convert-to-spot move))
+
+(defn draw?
+  [board]
+  (board/full? board))
+
+(defn matching-pattern?
+  [token pattern]
+  (loop [token token
+         pattern pattern]
+    (match pattern
+      [token] true
+      [" " & r] false
+      [(_ :guard #(not= token %)) & r] false
+      [token & r] (recur token (vec (rest pattern))))))
```

Comment:
Does order matter here? I am wondering if you can make the second to last clause to the end and remove the guard.

Reply:
[Rearranged it and removed the guard.](https://github.com/saralein/tic-tac-toe_clojure/pull/6/files#diff-f014d0e7db79dee6fc42af72f74d5325R19)